cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/model/Executable.java

```diff
@@ -23,6 +23,7 @@
  */
 public abstract class Executable {
 
+    public static final char DOT = '.';
```

Comment:
maybe we should extract common constants in classes in another module like `entities`